model/execution/implementation/Lti1p3ContextCacheRepository.php

```diff
@@ -38,6 +38,18 @@ public function __construct(?CacheItemPoolInterface $cache = null)
         $this->cache = $cache;
     }
 
+    public function findByDeliveryExecutionId($deliveryExecutionId): ?LtiLaunchData
+    {
+
+        $item = $this->cache->getItem($this->buildKey($deliveryExecutionId));
```

Comment:
are we always returning item object here?
Can you confirm it?

Reply:
Its just a same as its sibling findByDeliveryExecution() but uses delivery id instead of delivery object so its always returning object.